tests/unittests/tests-gcoap/Makefile.include

```diff
@@ -0,0 +1,8 @@
+USEPKG += nanocoap
+USEMODULE += gnrc_sock_udp
+
+USEMODULE += gcoap
+USEMODULE += gnrc_udp
+USEMODULE += gnrc_ipv6
```

Comment:
1. gcoap is necessary because we are testing that module. :-)
2. gnrc_ipv6 is needed at least due to code in gcoap's _event_loop(), which references GNRC_NETTYPE_IPV6. This dependency is my assumption. Is there a better alternative?
3. gnrc_udp is not needed, and I will fix that. It is an implicit (Makefile.dep) dependency for gcoap. Thanks for that.